Research/Research/RSDColor.swift

```diff
@@ -153,10 +153,10 @@ extension RSDColor {
     /// - parameter hexString:  An RGB color defined using a hex code.
     /// - returns: A color if the hex is valid.
     public convenience init?(hexString: String) {
-        if hexString == "#FFFFFF" {
+        if hexString == "#FFFFFF" || hexString == "white" {
             self.init(white: 1, alpha: 1)
         }
-        else if hexString == "#000000" {
+        else if hexString == "#000000" || hexString == "black" {
```

Comment:
...but here, "black" really *is* black? Is that going to lead to confusion?

Reply:
This came up b/c Figma uses hex values for everything *except* "white". I am assuming that if "black" is ever used inn Figma, it will have the same issue of being defined as "black" rather than "#000000" and being pro-active about fixing it now rather than crashing later. ¯\_(ツ)_/¯

Reply:
Oh, I created a tool to pull these values out of the Figma Doc and "white" caused things to crash. :(